docs/src/examples/components/TextArea/Types/TextAreaResizeExample.shorthand.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react'
+import { TextArea } from '@stardust-ui/react'
+
+const TextAreaExample = () => <TextArea resize="both" placeholder="Type here..." />
+
+export default TextAreaExample
```

Comment:
```suggestion
export default TextAreaExampleResize
```

Please also change filename